src/Router/Connect.tsx

```diff
@@ -0,0 +1,25 @@
+import { isArray } from "lodash"
+import React, { SFC } from "react"
+import { Container, ContainerType, Subscribe } from "unstated"
+
+interface ConnectProps {
+  to:
+    | ContainerType<any>
+    | Container<any>
+    | Array<ContainerType<any> | Container<any>>
+  children(...instances: any[]): React.ReactNode
+}
+
+export const Connect: SFC<ConnectProps> = ({ to, children }) => {
+  const _to = isArray(to) ? to : [to]
+
+  return (
+    <Subscribe to={_to}>
+      {({ state }) => {
```

Comment:
Wondering about the signature here, if `_to` is an array of different state objects to subscribe for (let's say a component subscribes to both the global app state and a state specific to its own use), I think this callback returns two objects, so `({ state }, filterState) => ...` kinda thing.

Not sure if that's handled here (or if that's the intent to be used that way).